Gopkg.toml

```diff
@@ -67,6 +67,11 @@
 [[constraint]]
   name = "github.com/kubernetes-csi/csi-lib-utils"
   version = "0.1.0"
+
+[[constraint]]
+  name = "k8s.io/klog"
+  revision = "8139d8cb77af419532b33dfa7dd09fbc5f1d344f"
+
```

Comment:
Why this constraint?

If there is a reason, explain it in a comment.

Please don't use the `Gopkg.toml` file to lock onto a specific revision. That's what the `Gopk.lock` file is for, with `dep ensure -update` as the mechanism for updating to a new version.